src/amo/components/AddonRecommendations/index.js

```diff
@@ -133,14 +105,12 @@ export class AddonRecommendationsBase extends React.Component<Props> {
     }
   }
 
-  cohort: CohortName;
-
-  dispatchFetchRecommendations({ guid, recommended }: Object) {
+  dispatchFetchRecommendations(guid: string) {
     this.props.dispatch(
       fetchRecommendations({
         errorHandlerId: this.props.errorHandler.id,
         guid,
-        recommended,
+        recommended: true,
```

Comment:
I think it would be nicer if this parameter defaulted to `true` in `fetchRecommendations()` so that the caller can leave it undefined.